data/json/recipes/electronics/tools.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "result": "makeshift_oscilloscope",
+    "type" : "recipe",
+    "category": "CSC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 2 ],
+    "time": 240000,
+    "book_learn": [ [ "advanced_electronics", 4 ], [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ] ],
+    "qualities": [
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "MULTIMETER", "level": 2 }
+    ],
+    "tools": [ [ [ "soldering_iron", 20 ], [ "toolset", 20 ] ] ],
+    "components": [
+      [ [ "television", 1 ] ],
```

Comment:
TV is just 12? I was pretty sure it was 20.

Well, it can be 12.

Though why use the entire TV when you could just use a small screen from a PDA or something like that?